python/cugraph/structure/graph.py

```diff
@@ -292,6 +293,7 @@ def from_cudf_edgelist(
         destination="destination",
         edge_attr=None,
         renumber=True,
+        store_transposed=False,
```

Comment:
The naming and effect of this flag can be confusing because we have been automatically storing both transposed and non-transposed (as needed) in SG path since cugraph's first release.
Perhaps `runmber_by=src/dst` would be clearer.

Reply:
@seunghwak and I have waffled on naming this parameter.  We could do this instead.  Might be a bit clearer.

Reply:
Before jumping to the naming, we need to make our strategy clear on storing both transposed and non-transposed (and actually edge list as well). This will largely cut the maximum graph size we can handle. I think we need to decide between "cuGraph becoming less magical" vs "largely reducing the maximum graph size cuGraph can handle".

Reply:
And if we want to enable users to fit larger graphs, we need to explicitly ask users to pick the format. ```renumber_by=src/dst``` is not explicit in this regard.